pipelines/cflinuxfs4.yml

```diff
@@ -11,390 +11,709 @@ params:
+  type: git
+  source:
+    uri: https://github.com/cloudfoundry/bosh-deployment.git
+    branch: master
+
+- name: cf-deployment
+  type: git
+  source:
+    uri: https://github.com/cloudfoundry/cf-deployment
+    branch: main
+
+- name: buildpacks-ci
+  type: git
+  source:
+    uri: https://github.com/cloudfoundry/buildpacks-ci
+    branch: master
+
+- name: cflinuxfs4
+  type: git
+  source:
+    branch: main
+    uri: git@github.com:cloudfoundry/cflinuxfs4.git
+    private_key: ((cflinuxfs4-deploy-key.private_key))
+
+- name: cflinuxfs4-github-tags
+  type: git
+  source:
+    uri: git@github.com:cloudfoundry/cflinuxfs4.git
+    private_key: ((cflinuxfs4-deploy-key.private_key))
+    tag_filter: "*"
+
+- name: cflinuxfs4-build-trigger
+  type: git
+  source:
+    uri: git@github.com:cloudfoundry/cflinuxfs4.git
+    private_key: ((cflinuxfs4-deploy-key.private_key))
+    ignore_paths:
+    - receipt.cflinuxfs4.x86_64
+    - README.md
+    - .gitignore
+
+- name: new-cves
+  type: git
+  source:
+    uri: git@github.com:cloudfoundry/public-buildpacks-ci-robots
+    paths: [ new-cve-notifications/ubuntu22.04.yml ]
+    private_key: ((public-buildpacks-ci-robots-deploy-key.private_key))
+    branch: main
+
+- name: receipt-diff
+  type: git
+  source:
+    uri: git@github.com:cloudfoundry/public-buildpacks-ci-robots
+    paths: [ receipt.cflinuxfs4.x86_64 ]
+    private_key: ((public-buildpacks-ci-robots-deploy-key.private_key))
+    tag_filter: "newpackages_cflinuxfs4_*"
+
+- name: public-robots
+  type: git
+  source:
+    uri: git@github.com:cloudfoundry/public-buildpacks-ci-robots
+    private_key: ((public-buildpacks-ci-robots-deploy-key.private_key))
+    branch: main
+
+- name: cflinuxfs4-release
+  type: git
+  source:
+    branch: main
+    uri: git@github.com:cloudfoundry/cflinuxfs4-release.git
+    private_key: ((cflinuxfs4-release-deploy-key.private_key))
+
+- name: capi-release
+  type: git
+  source:
+    uri: https://github.com/cloudfoundry/capi-release
+    branch: main
+
+- name: stack-s3
+  type: s3
+  source:
+    bucket: pivotal-buildpacks
+    regexp: rootfs/cflinuxfs4-(.*).tar.gz
+    access_key_id: ((pivotal-buildpacks-s3-access-key))
+    secret_access_key: ((pivotal-buildpacks-s3-secret-key))
+
+- name: receipt-s3
+  type: s3
+  source:
+    bucket: pivotal-buildpacks
+    regexp: rootfs/receipt.cflinuxfs4.x86_64-(.*)
+    access_key_id: ((pivotal-buildpacks-s3-access-key))
+    secret_access_key: ((pivotal-buildpacks-s3-secret-key))
+
+- name: cflinuxfs4-cf-deployment
+  type: bosh-deployment
+  source:
+    deployment: cf
+    skip_check: true
+
+- name: cflinuxfs4-rootfs-smoke-test-deployment
+  type: bosh-deployment
+  source:
+    skip_check: true
+    deployment: rootfs-smoke-test
+
+- name: gcp-stemcell
+  type: bosh-io-stemcell
+  source:
+    name: bosh-google-kvm-ubuntu-bionic-go_agent
+
+- name: cflinuxfs4-image
+  type: docker-image
+  source:
+    #! Later TODO: move to cloudfoundry for cosmetic purposes
```

Comment:
In the future it would be nice to have the image at `cloudfoundry/cflinuxfs4` following the pattern of fs3